.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
-node_modules/
\ No newline at end of file
+node_modules/
+coverage/
```

Comment:
Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding "files.insertFinalNewline": true to your settings.json file

Reply:
done